vagrant/roles/pxe-server/tasks/main.yml

```diff
@@ -0,0 +1,144 @@
+---
+- name: install required RPMs
+  package:
+    name: "{{item}}"
+    state: present
+  become: yes
+  with_items: "{{required_rpms}}"
+
+- name: create configuration file for eth1
+  template:
+    src: ifcfg-eth1.j2
+    dest: /etc/sysconfig/network-scripts/ifcfg-eth1
+  become: yes
+
+- name: restart network
+  service:
+    name: network
+    state: restarted
+  become: true
+
+- name: modify configuration file of dhcp server
+  template:
+    src: dhcpd.conf.j2
+    dest: /etc/dhcp/dhcpd.conf
+  become: yes
+
+- name: enable dhcpd service
+  service:
+    name: dhcpd
+    enabled: yes
+  become: true
+
+- name: start dhcpd service
+  service:
+    name: dhcpd
+    state: started
+  become: true
+
+- name: enable tftp.socket
+  service:
+    name: tftp.socket
+    enabled: yes
+  become: true
+
+- name: start tftp.socket
+  service:
+    name: tftp.socket
+    state: started
+  become: true
+
+- name: enable vsftpd service
+  service:
+    name: vsftpd
+    enabled: yes
+  become: true
+
+- name: start vsftpd service
+  service:
+    name: vsftpd
+    state: started
+  become: true
+
+- name: download minimal installation iso image to localhost
+  get_url:
+    url: "{{iso_img_url}}"
+    dest: vagrant_data
+    checksum: "{{iso_img_checksum}}"
+  delegate_to: localhost
+
+- name: create directory /vagrant_data
+  file:
+    path: /vagrant_data
+    state: directory
+    owner: vagrant
+    group: vagrant
+    mode: 0755
+  become: true
+
+- name: copy iso image to pxe server
+  copy:
+    src: vagrant_data
```

Comment:
Needs to end in `/`; otherwise, we end up with `/vagrant_data/vagrant_data/*.iso` instead of `/vagrant_data/*.iso`

Reply:
Yeah. I already noticed it.